test/Test_Menhir.re

```diff
@@ -14,11 +14,12 @@ let strip_Wrap_and_add_builtins =
         | Parens(e)
         | Probe(e, _) => cont(e)
         | Var(x) =>
-          let builtin = Util.VarMap.lookup(Builtins.Pervasives.builtins, x);
+          let builtin = Util.VarMap.lookup(Builtins.builtin_map, x);
```

Comment:
So it looks like some of the tests are breaking since the documentation buffers are shadowing the builtins (map).  We do the builtins based on the context which is shadowable and not in the parser ordinarily.

I think we should just stop parsing builtins as builtins in menhir. The original use case to mostly use it for elaboration tests has faded somewhat and the only test that fails if we parse them as vars is `Built-in function (menhir).`. If we end up needing a literal syntax for the builtins (which I hope we don't) we can add it back to menhir with special syntax.

Reply:
Let me know if you want me to make this change

Reply:
oh. i see. i though test-quick just did the same tests but... faster?

Reply:
If only. It skips the pbt and the longer tylr ones